src/utils/passportStrategyCallback.js

```diff
@@ -0,0 +1,31 @@
+import models from '../models';
+
+/**
+   * It uses the done method to return an authenticated user or an error object
+   * @function strategyCallback
+   * @param {string} accessToken
+   * @param {string} refreshToken
+   * @param {Object} profile
+   * @param {function} done
+   * @returns {Object} (user | error)
+*/
+const strategyCallback = async (accessToken, refreshToken, profile, done) => {
+  try {
+    if (!profile) return done(null, undefined);
+    const email = profile.emails[0].value;
+    const existingUser = await models.Users.findOne({ where: { email } });
+    if (existingUser) {
+      return done(null, existingUser);
+    }
+    const { name: { familyName: lastName, givenName: firstName } } = profile;
+    const password = Math.random().toString(36).slice(2);
```

Comment:
Create a password in the .env file and use the password here. Remember to hash the password. This is to maintain consistency.

Reply:
Done.